### DISCUSSION TITLE
version for database

### DISCUSSION THREAD

**Discussion by cristanchoa** [egeulgen/pathfindR#133] [General]
Hi, Thank you for the great package! I had run the same code a few months apart and got different enrichment results - not incredibly different, but enough to be noticeable. I am using Gene Ontology. I was wondering if it is an issue on how the terms are being pulled. Is it directly from their repository so it is the most up to date one or from the package itself? If so, is there a way to just consistently pull one version, perhaps by creating a custom list?

Thanks, 
Ana

**Comment by egeulgen** [comment]
Hello,
Unless custom gene sets are used, the terms within the `pathfindR.data` package are used. These are regularly (per ~6-12 months) updated. The table `pathfindR.data_updates` data frame contains all built-in data and when they were updated.

One other issue is that the algorithms for active subnetwork search are stochastic so slight differences in the results are expected.

Hope this helps,
-E

**Comment by cristanchoa** [comment]
Thank you! I was able to see the updates and looks like the most recent version was after my initial analysis. I have been using set.seed hoping that would help with any stochastic changes between runs. Is it possible that doesn't work with pathfindR? That would be fine, but I would work it into my pipeline.

Thanks again,
Ana


From: Ege Ulgen ***@***.***>
Sent: Sunday, July 24, 2022 9:34 AM
To: egeulgen/pathfindR ***@***.***>
Cc: Cristancho, Ana G ***@***.***>; Author ***@***.***>
Subject: [External]Re: [egeulgen/pathfindR] version for database (Discussion #133)


Hello,
Unless custom gene sets are used, the terms within the pathfindR.data package are used. These are regularly (per ~6-12 months) updated. The table pathfindR.data_updates data frame contains all built-in data and when they were updated.

One other issue is that the algorithms for active subnetwork search are stochastic so slight differences in the results are expected.

Hope this helps,
-E

-
Reply to this email directly, view it on GitHub<https://nam10.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fegeulgen%2FpathfindR%2Fdiscussions%2F133%23discussioncomment-3216169&data=05%7C01%7CCristanchoA%40chop.edu%7C9f2d38e78b454dac2ba108da6d792856%7Ca611241607b041a59bb1d146b575c975%7C0%7C0%7C637942664385311625%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=jXSKhkxIQaxKUsQIWskmUX4tQu7fgtLIvQhB5IGU9Aw%3D&reserved=0>, or unsubscribe<https://nam10.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FASPDMCDD2XTXOYJULSAFV2DVVVA4FANCNFSM54JD6M7A&data=05%7C01%7CCristanchoA%40chop.edu%7C9f2d38e78b454dac2ba108da6d792856%7Ca611241607b041a59bb1d146b575c975%7C0%7C0%7C637942664385311625%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=pC4oI4okIHI2xTmyhqqljkCj5WYvL32F%2F6xQHvY1eeY%3D&reserved=0>.
You are receiving this because you authored the thread.Message ID: ***@***.******@***.***>>
** This email originated from an EXTERNAL sender to CHOP. Proceed with caution when replying, opening attachments, or clicking links. Do not disclose your CHOP credentials, employee information, or protected health information to a potential hacker**.

**Comment by egeulgen** [comment]
Hello again, unfortunately set.seed does not work at the moment (because the active subnetwork search is implemented in java) but we are working on this and I will let you know
